From 7b3f79b80e45c5eed096ff9ded96912980f7e66e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?D=C3=A1niel=20J=C3=A1rai?= <jaraidaniel@gmail.com>
Date: Wed, 5 Apr 2017 19:27:03 +0100
Subject: [PATCH 5/5] msm8960: define HFP_ASM_RX_TX to fix build breakage

For the time being, use the msm8974 value.
This change should be reverted once a proper
solution is found.

Change-Id: Iad426550df335bfcf04db6954a40bfb8fd5725a7
---
 hal/msm8960/platform.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hal/msm8960/platform.h b/hal/msm8960/platform.h
index de74563..51f75d2 100644
--- a/hal/msm8960/platform.h
+++ b/hal/msm8960/platform.h
@@ -131,5 +131,6 @@ enum {
 #define AFE_PROXY_PLAYBACK_PCM_DEVICE 7
 #define AFE_PROXY_RECORD_PCM_DEVICE 8
 
+#define HFP_ASM_RX_TX 24 // temporary
 
 #endif // QCOM_AUDIO_PLATFORM_H
-- 
2.7.4

